pkg/scannerv4/mappers/mappers.go

```diff
@@ -665,20 +664,28 @@ func cvssMetrics(_ context.Context, vuln *claircore.Vulnerability, nvdVuln *nvds
 	case strings.HasPrefix(vuln.Updater, osvUpdaterPrefix) && !isOSVDBSpecificSeverity(vuln.Severity):
 		preferredCVSS, preferredErr = vulnCVSS(vuln, v4.VulnerabilityReport_Vulnerability_CVSS_SOURCE_OSV)
 	case strings.EqualFold(vuln.Updater, constants.ManualUpdaterName):
-		// It is expected manually added vulnerabilities only have a single link.
-		preferredCVSS, preferredErr = vulnCVSS(vuln, sourceFromLinks(vuln.Links))
+		src := sourceFromLinks(vuln.Links)
+		// When both NVD and manual data have available NVD CVSS scores, use the NVD score.
+		// And only the NVD score is included in the metrics, as the manual data originates from the same
+		if src == v4.VulnerabilityReport_Vulnerability_CVSS_SOURCE_NVD {
+			preferredCVSS, preferredErr = nvdCVSS(&nvdVuln)
```

Comment:
Let's make this just `nvdCVSS(nvdVuln)` and make `nvdCVSS` take in a `nvdschema.CVEAPIJSON20CVEItem` instead of a pointer to it since `nvdCVSS` does not account for `nil`